### PR TITLE
Adding the ability to set tolerations for the webhook deployment

### PR DESCRIPTION
**Pull request by MbolotSuse**

### CODE REVIEW

File: charts/rancher-webhook/templates/deployment.yaml [hunk 1]
```diff
@@ -18,6 +18,10 @@ spec:
       {{- if .Values.global.hostNetwork }}
       hostNetwork: true
       {{- end }}
+      {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
```
Comment (genexpr) [inline charts/rancher-webhook/templates/deployment.yaml:23]: Is the leading dash before `toYaml` needed?

Comment (MbolotSuse) [reply charts/rancher-webhook/templates/deployment.yaml:23]: It's not strictly necessary (chart still renders) but there's some extra whitespace that I'd rather not have. It's the difference between:

(w/o dash)
```
      tolerations:

        - effect: NoSchedule
          key: test1
          operator: Equals
          value: test1
```
and 

(with dash)
```
      tolerations:
        - effect: NoSchedule
          key: test1
          operator: Equals
          value: test1
```

**Comment by MbolotSuse** [comment]
Part of rancher/rancher#34159 .

**Review by rmweir — APPROVED** [review]
LGTM

**Pull request merged**